DigitalLearningSolutions.Data.Migrations/Scripts/UAR-831-CreateViewsForAdminUsersAndCandidatesTables-UP.sql

```diff
@@ -1,45 +1,43 @@
 ﻿CREATE VIEW AdminUsers AS
-SELECT dbo.Users.ID                                      AS AdminID,
-       dbo.AdminAccounts.Login_deprecated                AS Login,
-       dbo.Users.PasswordHash                            AS Password,
+SELECT dbo.AdminAccounts.AdminID,
+       null                                    AS Login,
+       dbo.Users.PasswordHash                  AS Password,
        dbo.AdminAccounts.CentreID,
-       dbo.AdminAccounts.IsCentreAdmin                   AS CentreAdmin,
-       dbo.AdminAccounts.ConfigAdmin_deprecated          AS ConfigAdmin,
-       dbo.AdminAccounts.IsReportsViewer                 AS SummaryReports,
-       dbo.AdminAccounts.IsSuperAdmin                    AS UserAdmin,
-       dbo.Users.FirstName                               AS Forename,
-       dbo.Users.LastName                                AS Surname,
-       dbo.AdminAccounts.Email,
+       dbo.AdminAccounts.IsCentreAdmin         AS CentreAdmin,
+       0                                       AS ConfigAdmin,
+       dbo.AdminAccounts.IsReportsViewer       AS SummaryReports,
+       dbo.AdminAccounts.IsSuperAdmin          AS UserAdmin,
+       dbo.Users.FirstName                     AS Forename,
+       dbo.Users.LastName                      AS Surname,
+       dbo.Users.PrimaryEmail                  AS Email,
        dbo.AdminAccounts.IsCentreManager,
-       dbo.AdminAccounts.Approved_deprecated             AS Approved,
-       dbo.AdminAccounts.PasswordReminder_deprecated     AS PasswordReminder,
-       dbo.AdminAccounts.EITSProfile_deprecated          AS EITSProfile,
-       dbo.AdminAccounts.PasswordReminderHash_deprecated AS PasswordReminderHash,
-       dbo.AdminAccounts.PasswordReminderDate_deprecated AS PasswordReminderDate,
+       true                                    AS Approved,
+       0                                       AS PasswordReminder,
+       ""                                      AS EITSProfile,

```

Comment:
I think these are the wrong type of quote marks as I am getting and IDE error here. Should be `''` for SQL